prototype/pca_compression/pca_compression.ino

```diff
@@ -0,0 +1,101 @@
+#define REAL_NUMBER float
+#define INTEGER uint8_t
+#define DIGITS_PRECISION 3
+#define MATRIX_SIZE 8
```

Comment:
Don't we receive the matrix size or the number of elements through the serial port?